src/test/groovy/it/move2/agent/application/UpdaterTest.groovy

```diff
@@ -0,0 +1,62 @@
+package it.move2.agent.application
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import it.move2.agent.configuration.PublisherConfiguration
+import it.move2.agent.ports.JobBoard
+import it.move2.agent.ports.Queue
+import spock.lang.Specification
+import spock.lang.Subject
+
+class UpdaterTest extends Specification {
+    private JobBoard jobBoard1
+    private JobBoard jobBoard2
+    private Queue queue
+    private ObjectMapper objectMapper
+    private Publisher publisher
+
+    @Subject
+    private Updater updater
+
+    def setup() {
+        jobBoard1 = Mock()
+        jobBoard2 = Mock()
+        queue = Mock()
+        objectMapper = new ObjectMapper()
```

Comment:
How about using the same ObjectMapper that is used in the code from the Bootstrap class? Currently, this ObjectMapper is configured a bit different in tests than in the production version, which may not reveal all the bugs.